### ISSUE TITLE
PlantUML rendering hangs on Windows 10 

### ISSUE THREAD

**Issue by JoostvanPinxten** [asciidoctor/asciidoctor-diagram#255]
After installing the latest (2.0.0) Asciidoctor-diagram, asciidoctor does not produce output anymore, and the terminal hangs indefinitely. The previously installed version (1.5.18) did produce output for our cases. A fresh install with 2.0.0 asciidoctor-diagram on a different Windows machine yielded the same results.

After some debugging, we found out that the reason seems to be the conversion of our PlantUML drawings. I.e., the following .adoc cannot be succesfully converted:

```
[plantuml]
....

....
``` 

```
$ asciidoctor -r asciidoctor-diagram test.adoc
```

System info:
Windows 10 Pro, build 1903

```
$ asciidoctor --version
Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.6.4p104 (2019-08-28 revision 67798) [x64-mingw32]) (lc:IBM437 fs:Windows-1252 in:IBM437 ex:IBM437)
```

```
$ gem list 
asciidoctor (2.0.10)
asciidoctor-diagram (2.0.0, 1.5.18)
bigdecimal (default: 1.4.1)
bundler (default: 1.17.2)
cmath (default: 1.0.0)
csv (default: 3.0.9)
date (default: 2.0.0)
dbm (default: 1.0.0)
did_you_mean (1.3.0)
e2mmap (default: 0.1.0)
etc (default: 1.0.1)
fcntl (default: 1.0.0)
fiddle (default: 1.0.0)
fileutils (default: 1.1.0)
forwardable (default: 1.2.0)
gdbm (default: 2.0.0)
io-console (default: 0.4.7)
ipaddr (default: 1.2.2)
irb (default: 1.0.0)
json (default: 2.1.0)
logger (default: 1.3.0)
matrix (default: 0.1.0)
minitest (5.11.3)
mutex_m (default: 0.1.0)
net-telnet (0.2.0)
openssl (default: 2.1.2)
ostruct (default: 0.1.0)
power_assert (1.1.3)
prime (default: 0.1.0)
psych (default: 3.1.0)
rake (12.3.2)
rdoc (default: 6.1.2)
rexml (default: 3.1.9)
rss (default: 0.2.7)
scanf (default: 1.0.0)
sdbm (default: 1.0.0)
shell (default: 0.7)
stringio (default: 0.0.2)
strscan (default: 1.0.0)
sync (default: 0.5.0)
test-unit (3.2.9)
thwait (default: 0.1.0)
tracer (default: 0.1.0)
webrick (default: 1.4.2)
xmlrpc (0.3.0)
zlib (default: 1.0.0)
```

**Comment by pepijnve** [comment]
Thanks for reporting this. #254 covers the same problem. This was fixed yesterday and the fix has been shipped as part of release 2.0.1. Upgrading should resolve the problem.

**Comment by JoostvanPinxten** [comment]
Fantastisch! Thanks for fixing it. 

Too bad we parked it before the other bug was filed, and I forgot to check the closed bugs again...
